crates/dojo-test-utils/src/anvil.rs

```diff
@@ -0,0 +1,45 @@
+use std::time::Duration;
+
+use ethers::middleware::SignerMiddleware;
+use ethers::providers::{Http, Provider};
+use ethers::signers::{LocalWallet, Signer};
+use ethers::utils::AnvilInstance;
+use ethers_core::utils::Anvil;
+
+pub struct TestAnvil {
+    url: String,
+    pub anvil: AnvilInstance,
```

Comment:
ah i didnt know `anvil` has this. we should do the same for `katana`, separating the cli instance type and the runner.